### PR TITLE
chore(component-testing): CT-141 update project-ct

### PR DESCRIPTION
**Pull request by lmiller1990**
Just move to TS, no change in behavior.

### CODE REVIEW

File: packages/server-ct/src/project-ct.ts [hunk 1]
```diff
@@ -263,7 +274,7 @@ class Project {
   //   return this.watchers.watch(settings.pathToCypressEnvJsthis.projectRoot), obj)
   // }
 
-  watchSettingsAndStartWebsockets (options = {}, cfg = {}) {
+  watchSettingsAndStartWebsockets (options: Record<string, any> = {}, cfg: Record<string, any> = {}) {
```
Comment (dmtrKovalenko) [inline packages/server-ct/src/project-ct.ts:277]:
```suggestion
  watchSettingsAndStartWebsockets (options: Record<string, unknown> = {}, cfg: Record<string, unknown> = {}) {
```

File: packages/server-ct/src/project-ct.ts [hunk 2]
```diff
@@ -279,12 +290,14 @@ class Project {
 
       onSavedStateChanged: options.onSavedStateChanged,
 
-      onCaptureVideoFrames: (data) => {
+      onCaptureVideoFrames: (data: any) => {
```
Comment (dmtrKovalenko) [inline packages/server-ct/src/project-ct.ts:293]:
```suggestion
      onCaptureVideoFrames: (data: unknown) => {
```

**Comment by dmtrKovalenko** [comment]
Would be better to use `unknown` for types in order to force the correct type signature in the future

**Comment by lmiller1990** [comment]
@dmtrKovalenko nice, thanks, good idea

**Pull request merged**